tox.ini

```
@@ -9,13 +9,13 @@ envlist = fmt, lint, unit
 [vars]
 src_path = {toxinidir}/src/
 tst_path = {toxinidir}/tests/
-;lib_path = {toxinidir}/lib/charms/operator_name_with_underscores
-all_path = {[vars]src_path} {[vars]tst_path} 
+lib_path = {toxinidir}/lib/charms/
```

I think it's best to change this to 
```suggestion
lib_path = {toxinidir}/lib/charms/identity_platform_login_ui
```

This is only used for calculating coverage and I don't think we need to calculate our test coverage for libraries that belong to other charms.